examples/gatsby/README.md

```diff
@@ -16,6 +16,12 @@ npm install
 npm run develop
 ```
 
+## Providing the theme
+
+To propagate the theme to a component tree use the `src/withRoot.js` HOC.
+You should wrap **only** your top-level components with this HOC otherwise you risk re-rendering your React tree multiple times and styling issues during the build phase.
```

Comment:
"and causing styling issues"?

Reply:
I provided some screenshots on the #12649 . dont know how to describe best the issues in the screenshots.